client/reader/following-manage/index.jsx

```diff
@@ -198,7 +204,7 @@ class FollowingManage extends Component {
 							/>
 						</div> }
 				</div>
-				{ ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
+				{ hasFollows && ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
```

Comment:
though we could always spin that up in a new PR. I think this is good enough to land.